src/subscribables/dependentObservable.js

```diff
@@ -440,9 +440,14 @@ ko.isComputed = function (instance) {
     return ko.hasPrototype(instance, ko.computed);
 };
 
+ko.isPureComputed = function (instance) {
+    return (instance && instance._state && instance._state.pure && ko.isComputed(instance)) || false;
```

Comment:
Awesome. Maybe it can condense it with e.g.

```javascript
ko.isPureComputed = function(instance) {
   return ko.hasPrototype(instance, ko.computed) &&
     instance.getVersion === pureComputedOverrides.getVersion
}
```

This avoids the overhead of an extra function call to `ko.isComputed`, and we can get rid of the extra `!instance` test since it's subsumed by `hasPrototype`.